libsolidity/ast/ASTAnnotations.h

```diff
@@ -232,6 +232,12 @@ struct TypeNameAnnotation: ASTAnnotation
 	TypePointer type = nullptr;
 };
 
+struct IdentifierPathAnnotation: TypeNameAnnotation
+{
+	/// Referenced declaration, set during reference resolution stage.
+	Declaration const* referencedDeclaration = nullptr;
+};
+
 struct UserDefinedTypeNameAnnotation: TypeNameAnnotation
 {
 	/// Referenced declaration, set during reference resolution stage.
```

Comment:
Is this still needed?

Reply:
Well, I suppose one could also use the `IdentifierPathAnnotation` instead